GitUI/Avatars/GithubAvatarProvider.cs

```diff
@@ -50,7 +50,7 @@ public async Task<Image> GetAvatarAsync([NotNull] string email, string name, int
             // GitHub are never scaled and always 420 x 420 - even if a different size was requested.
 
             // We exploit that fact to filter out identicons.
-            var isIdenticon = imageSize != 420 && image.Size.Width == 420;
+            var isIdenticon = imageSize != 420 && image != null && image.Size.Width == 420;
```

Comment:
Bug fix. The download can fail and return a null `image` which would have thrown here in the past.